docs/reference.rst

```diff
@@ -133,4 +133,65 @@ API Reference
     .. py:method:: none()
         :classmethod:
 
-        Returns an instance which allows for no drift.
```

Comment:
Just curious why this line got removed and replaced by the same line? Some whitespace change?

Reply:
I think it's just git doing a sub-optimal diff.